### ISSUE TITLE
Mailinglist

### ISSUE THREAD

**Issue by certik** [spacetelescope/asdf-standard#123]
What forum should be used to discuss the format? Is there a mailinglist? It would be nice to clarify in the README how people can get involved in discussions and development of the format.

**Comment by justincely** [comment]
Hi @certik, thanks for pointing this out.  We'll get a proper venue for discussing this setup shortly, and i'll update this issue and the README with the details.

**Comment by certik** [comment]
Thanks @justincely, I appreciate that.

**Comment by justincely** [comment]
@certik, after much round-about (sorry for the delay) we've decided to stick with the github issues for discussions around all things ASDF.  We played with some additional options, but these had the disadvantage of not living close to the code, not being fully ready to deploy, or having higher bars of entry for the community. 

So for now, we'll keep all discussion here - and if the community grows large enough to need another method we can move at that point.

I've updated the readme to mention contributions, feedback, ideas should come from the issue tracker.
